### ISSUE TITLE
Dependency problem with Debian package: Package libopencv-core3.2 is not installed/available

### ISSUE THREAD

**Issue by sesselastronaut** [PetteriAimonen/focus-stack#13]
Dear Petteri Aimonen,
I encounter a dependency problem installing the pre-compiled version as well as a self-compiled version. When installing the package it returns: `Package libopencv-core3.2 is not installed` 
This is on Debian Buster with only `libopencv-core4.5/testing,now 4.5.1+dfsg-4 amd64` available from the resource. Any idea how this dependency can be satisfied much appreciated.

**Comment by PetteriAimonen** [comment]
Sounds like same problem as #10

**Comment by sesselastronaut** [comment]
Thanks for that indication @PetteriAimonen , unfortunately changing the line `Depends: libopencv-core3.2|libopencv-core4.5, libopencv-video3.2|libopencv-video4.5, libopencv-imgcodecs3.2|libopencv-imgcodecs4.5, libopencv-photo3.2|libopencv-photo4.5, libopencv-imgproc3.2|libopencv-imgproc4.5, ${shlibs:Depends}, ${misc:Depends}, libgcc1, libstdc++6` as stated in #10 (adapted to the versions on my machine) doesn't change the dependency problem. Still getting the following error when executing `sudo dpkg -i DEBUILD/focus-stack*.deb` on the self-compiled version:
```

dpkg: dependency problems prevent configuration of focus-stack:
 focus-stack depends on libopencv-core3.2; however:
  Package libopencv-core3.2 is not installed.
 focus-stack depends on libopencv-video3.2; however:
  Package libopencv-video3.2 is not installed.
 focus-stack depends on libopencv-imgcodecs3.2; however:
  Package libopencv-imgcodecs3.2 is not installed.
 focus-stack depends on libopencv-photo3.2; however:
  Package libopencv-photo3.2 is not installed.
 focus-stack depends on libopencv-imgproc3.2; however:
  Package libopencv-imgproc3.2 is not installed.
```

I'm not too familiar with Debian packaging and compiling. I tried the other version as stated in #10 `Depends: libc6 (>= 2.14), libgcc-s1 (>= 3.0), libopencv-core3.2 (>= 3.2.0+dfsg) | libopencv-core4.5 (>= 4.5.1+dfsg), libopencv-imgcodecs3.2 (>= 3.2.0+dfsg) | libopencv-imgcodecs4.5 (>= 4.5.1+dfsg), libopencv-imgproc3.2 (>= 3.2.0+dfsg) | libopencv-imgproc4.5 (>= 4.5.1+dfsg), libopencv-video3.2 (>= 3.2.0+dfsg) | libopencv-video4.5 (>= 4.5.1+dfsg), libstdc++6 (>= 6), libgcc1` but same result here.

**Comment by PetteriAimonen** [comment]
Yeah. Perhaps try building with plain `make`, even though that won't give a nice debian package.

**Comment by sesselastronaut** [comment]
Ja I did that and it works fine :) Thanks again, also for providing such a nice piece of software!

**Comment by PetteriAimonen** [comment]
:)

I hope I'll eventually find a fix for the debian packaging also, I just haven't had time to study how it is supposed to work.

**Comment by sesselastronaut** [comment]
If it's ok for you I would leave the issue open, maybe someone with Debian packaging knowledge will stumble upon it and is able to help

**Comment by PetteriAimonen** [comment]
Yeah that is ok.
